jmx-config.yml.k8s_sample

```diff
@@ -20,6 +20,7 @@
           JMX_USER: admin
           JMX_PASS: admin
           COLLECTION_FILES: ${config.path}
+          REMOTE_MONITORING: true
```

Comment:
can you please add the same description as we have on other integrations?

Reply:
sure